.gitignore

```diff
@@ -22,7 +22,10 @@
 # Ignore uploaded files in development.
 /storage/*
 !/storage/.keep
+
+# Ignore local development history/config
 .byebug_history
+.ruby-version
```

Comment:
We might want to consider adding these to your global gitignore. Maybe a playbook-worthy thing, since this requires all devs to use similar practices?

Reply:
ah, fair point.  created my own global gitignore and added `.ruby-version` to it.  then removed this change from this PR.

i'll probably refrain from a playbook PR... i think i've added `.ruby-version` to `.gitignore` in other projects, and _maybe_ have seen others do that, but i'm not sure, and i definitely don't recall anyone else complaining about it showing up in `git status` output.  so i'll be lazy and wait to see if anyone else cares enough to put up a similar PR, or bring up the topic in dev planning, or whatever 🙂

Reply:
OK, sounds good!